src/pages/index.jsx

```diff
@@ -3,11 +3,13 @@ import Container from "@mui/material/Container";
 import CssBaseline from "@mui/material/CssBaseline";
 import React from "react";
 import theme from "../theme/theme";
```

Comment:
We should separate local and global import for better readability

also in other places